src/compiler/checker.ts

```diff
@@ -11177,6 +11183,13 @@ namespace ts {
                 });
             }
 
+            if (multipleDefaultModifiers) {
+                ts.forEach(declarations, function (declaration) {

```

Comment:
Use a `for-of`. Additionally, we almost always use arrow functions instead of traditional functions.